deploy/openshift/templates/monitoring/che-monitoring.yaml

```diff
@@ -83,8 +83,10 @@ objects:
           volumeMounts:
           - mountPath: /etc/grafana/provisioning/datasources
             name: volume-datasources
-          - mountPath: /etc/grafana/provisioning/dashboards
+          - mountPath: /grafana-dashboard-definitions/0
```

Comment:
is `/grafana-dashboard-definitions/0` expected `mountPath` ?

Reply:
I chose that mount path because that is what downstream uses, [grafana docs default ](https://grafana.com/docs/administration/provisioning/#dashboards)is in `/etc/grafana/provisioning/dashboards`, but that is where we mount the `grafana-dashboard-provider`

Reply:
kind of strange. Can you explain why we need that in upstream?

Reply:
The convention of Hosted Che is to place the dashboard models in `/grafana-dashboard-definitions`.  `/etc/grafana/provisioning/dashboards` is taken as a mount point by the dashboard provider configuration.  I put this change in upstream to make upstream Che look more like Hosted Che.

Alternatively, I could change the mount path, or place everything in `/etc/grafana/provisioning/dashboards` using `subPath`, and tell the provider configuration to look in the `subPath` for the dashboard models.

Reply:
I think in the case of che-monitoring.yaml we should be close to what vanilla Grafana documentation recommends. @ibuziuk wdyt?

Reply:
@ibuziuk is on PTO for the next two weeks.  We could put the dashboard in e.g. `/etc/grafana/provisioning/dashboards/che`, but that would be against the goal of being able to run the same configmaps upstream and downstream without changing them.

The mount path of the dashboards IMO is an implementation detail that users shouldn't really have to care about.  They should just know that when they deploy the monitoring stack, they will see the dashboards that they are supposed to see.

Reply:
From what we discussed with @ibuziuk the goal was to make ```grafana-dashboards.yaml``` as close to downstream as possible. Other configuration should be close to Grafana upstream in my opinion.

Reply:
That makes sense.  I made a change to `che-monitoring.yaml` to reflect this.